langchain-llamaindex-slackbot/README.md

```diff
@@ -77,7 +77,7 @@ After the installation is completed, you directly run the pipeline locally
 right away.
 
 ```bash
-python run.py
+python src/run.py
```

Comment:
Why is this necessary now? Or was this always broken?

Reply:
Was always broken / incorrectly documented. Only came across it while redeploying.